ui/components/multichain/pages/send/send.test.js

```diff
@@ -250,7 +251,14 @@ describe('SendPage', () => {
   });
 
   describe('footer buttons', () => {
-    const mockStore = configureMockStore([thunk])(mockState);
+    const state = {
+      ...mockState,
+      metamask: {
+        ...mockState.metamask,
+        permissionHistory: {},
```

Comment:
Should this field be added to `mockState` itself?

Reply:
Yep, only needed one specific spot to fix.